lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -172,6 +174,11 @@ const FeatureInfoSection = React.createClass({
                             <When condition={reactInfo.showRawData || !this.hasTemplate()}>
                                 <If condition={reactInfo.hasRawData}>
                                     {reactInfo.rawData}
+                                    <If condition={reactInfo.timeSeriesChart}>
+                                        <div className="time-series-chart">
```

Comment:
ah, good question. No, this is an arbitrary class name ostensibly to allow user-specific styling, but which is really needed so that React gives this a different key to anything else in this section.